src/api/app/views/layouts/webui/responsive_ux/_unread_notifications_counter.html.haml

```diff
@@ -1,4 +1,4 @@
-= link_to(my_notifications_path, class: 'nav-link text-light p-0 w-100', alt: 'Notifications') do
+.notifications-counter
```

Comment:
The link isn't in this partial anymore since the CSS classes are slightly different depending on where we render this partial, right? I'm asking to be sure I understand the reasoning behind this change.

Reply:
Yep, this is because their classes differs a bit.